### PR TITLE
code cleaning

### PR DESCRIPTION
**Pull request by francescobrivio**
Code cleaning
Commented the following branches:
- NBadMu
- daughters_iseleBDT
- daughters_passConversionVeto
- daughters_eleMissingHits
- daughters_eleMissingLostHits
- daughters_SCeta
- daughters_L3FilterFired
- daughters_L3FilterFiredLast
- daughters_isL1IsoTau28Matched
- jets_rawPt

For NBadMu I also removed the relative modules in HiggsTauTauProducer_94X.py (this was used only for 2016 data ReReco).
For the electron related a cleaning could also be done in EleFille.cc to remove the related userfloats that have become unused.
For the trigger L3 filters I checked that they are not used in KLUB, but @camendola please give it a second look.
"isL1IsoTau28Matched" is never filled since we implemented the L1 taus and jets.
Also "jets_rawPt" is unused in KLUB, but @camendola have a look here as well.

### CODE REVIEW

**Comment by francescobrivio** [comment]
I still have to look at more possible braches to be commented out. @camendola once we double check we can delete completely these branches and maybe clean out some more commented stuff that is not used/useful anymore.

**Comment by francescobrivio** [comment]
also, before closing this PR, we should test some ntuples on KLUB just to make sure everything works fine

File: NtupleProducer/plugins/HTauTauNtuplizer.cc [hunk 1]
```diff
@@ -614,8 +614,8 @@ class HTauTauNtuplizer : public edm::EDAnalyzer {
   std::vector<Long64_t> _daughters_trgMatched;
   std::vector<Long64_t> _daughters_FilterFired;
   std::vector<Long64_t> _daughters_isGoodTriggerType;
-  std::vector<Long64_t> _daughters_L3FilterFired;
-  std::vector<Long64_t> _daughters_L3FilterFiredLast;
+  //std::vector<Long64_t> _daughters_L3FilterFired; //FRA January2019
+  //std::vector<Long64_t> _daughters_L3FilterFiredLast; //FRA January2019
```
Comment (camendola) [inline NtupleProducer/plugins/HTauTauNtuplizer.cc]: I'm always confused about these flags; they are not actually used, but I think we can keep them for debug

Comment (francescobrivio) [reply NtupleProducer/plugins/HTauTauNtuplizer.cc]: agree, I will add them back

**Comment by francescobrivio** [comment]
in the second commit ("code cleaning: more unused branches commented") I commented more unused branches:
- PUreweight: is already implemented in KLUB and was very out of date anyway
- PUPPIMet and PFMETCov: they are never used in KLUB, @l-cadamuro could you confirm this? or do you think there is a good reason to keep them?
- MC_weight_scale_muF0p5/muF2/muR0p5/muR2 : they are never used in KLUB, @l-cadamuro can you confirm these are obsolete?
- daughters_iseleCUT: this was already never filled, it has anyway been replaced with more up to date instructions
- daughters_LFtrigger and daughters_L3trigger: these were never filled
- jets_VBFfirstTrigMatch and jets_VBFsecondTrigMatch: these were the first implementation of VBF trigger matching, now obsolete, thus I removed them
- pv_x/y/z, pvRefit_x/y/z and isRefitPV: I enclosed the branches in a "if" so that the branches are added only if the variables are computed

**Comment by francescobrivio** [comment]
Following @camendola and @l-cadamuro 's suggestions I uncommented back some useful branches:
- MC_weight_scale_muF0p5/muF2/muR0p5/muR2: useful for estimating theory errors
- daughters_L3FilterFired/daughters_L3FilterFiredLast: for debugging purposes
- PUPPIMet: to test alternative met
- PFMetCov: used in SVfit calculation

**Pull request merged**